tests/bears/BearTest.py

```diff
@@ -119,6 +140,18 @@ def tearDown(self):
 
 class BearTest(BearTestBase):
 
+    def test_default_aspects(self):
```

Comment:
Why we don't have test if Bear is assigned an aspect correctly?
`assert AspectsBear.aspects['detect'] == [CommitMessage.Shortlog.ColonExistence]`